WalletWasabi.Backend/Global.cs

```diff
@@ -33,10 +34,18 @@ public Global(string dataDir, IRPCClient rpcClient, Config config, IHttpClientFa
 		HostedServices = new();
 		CoinVerifierHttpClient = WasabiHttpClientFactory.CreateLongLivedHttpClient();
 		HttpClientFactory = httpClientFactory;
+		NostrKeyManager = new(DataDir);
 
 		CoordinatorParameters = new(DataDir);
 		CoinJoinIdStore = CoinJoinIdStore.Create(CoordinatorParameters.CoinJoinIdStoreFilePath);
 
+		// Add Nostr publisher
+		// It is only supported on Mainnet.
+		if (Config.Network == Network.Main)
+		{
+			HostedServices.Register<CoordinatorNostrPublisher>(() => new CoordinatorNostrPublisher(TimeSpan.FromMinutes(15), NostrKeyManager.Key, new GingerNostrCoordinator()), "Coordinator Nostr Publisher");
+		}
```

Comment:
Do not run this PR on mainnet, otherwise it will send the message out and it will appear to the public.